src/briefcase/integrations/linuxdeploy.py

```diff
@@ -127,3 +128,102 @@ def patch_elf_header(self):
                 # The original or patched value. If this is the case, the file is likely
                 # wrong and we should raise an exception.
                 raise CorruptToolError("linuxdeploy")
+
+
+class LinuxDeploy(LinuxDeployBase):
+    name = "linuxdeploy"
+    full_name = "linuxdeploy"
+
+    @property
+    def file_path(self):
+        return self.command.tools_path / self.filename
+
+    @property
+    def filename(self):
+        return f"linuxdeploy-{self.command.host_arch}.AppImage"
+
+    @property
+    def linuxdeploy_download_url(self):
+        return (
+            "https://github.com/linuxdeploy/linuxdeploy/"
+            f"releases/download/continuous/{self.filename}"
+        )
+
+    def uninstall(self):
+        """Uninstall linuxdeploy."""
+        with self.command.input.wait_bar("Removing the old linuxdeploy install..."):
+            self.file_path.unlink()
+
+
+class LinuxDeployPluginBase(LinuxDeployBase):
+    """Base class for linuxdeploy plugins."""
+
+    @property
+    def file_path(self):
+        return self.command.tools_path / "linuxdeploy_plugins" / self.filename
```

Comment:
Ok - this is going to be problematic. It works for "known" plugins like GTK and Qt, because there's a known canonical version; but it won't work for user plugins, because two different projects could have a plugin with the same name. 

I suspect that we will be able to locally cache for "known" plugins - but anything user provided will need to be downloaded/copied directly into the project folder.